pkg/blockdevice/lvm.go

```diff
@@ -140,12 +140,22 @@ func (l *Lvm) Destroy(ctx context.Context) (bool, error) {
 	destroyed, err := l.LogicalVolume.Remove(ctx, l.CommandArgs.LvArgs.Remove)
 	if err != nil {
 		return false, err
-
 	}
 	if destroyed {
 		objectDestroyed = true
 	}
 
+	// Check to ensure the VG has no LVs before removing
+	lvs, err := lvm.LvsListVolumes(ctx, l.VolumeGroup.Log)
+	if err != nil {
+		return false, err
+	}
+	for _, lv := range lvs {
+		if lv.VGName == l.VolumeGroup.Name {
+			return false, nil
```

Comment:
Should this be true since it destroyed the LV?

Reply:
To me, hitting this line means that there is another LV that is still using the VG. This LV is now gone, but there are still others. 

Do I have that wrong?

Reply:
Ah, I see what you're saying now. I'll change that.

This got me thinking though: if we remove all the LVs and then somehow on the last one we don't remove the VG because `lvs` is stale or something. A. is that possible and B. are we going to leave around a VG in that scenario?